docs/modules/reference/pages/configuration/ssl.adoc

```diff
@@ -0,0 +1,384 @@
+
+We summarize here just in case this web page goes away.
+We assume the private key is in key.pem and the cert is in cert.pem (both are in PEM format).
+
+=== Convert the key and cert from PEM format to DER format using openssl command
+
+Use openssl to convert from PEM to DER format.
+
+[source, console]
+----
+openssl pkcs8 -topk8 -nocrypt -in key.pem -inform PEM -out key.der -outform DER
+openssl x509 -in cert.pem -inform PEM -out cert.der -outform DER
+----
+
+=== Put key and cert into a new Java Keystore
+
+Use the [ImportKey.java](http://www.agentbob.info/agentbob/80/version/default/part/AttachmentData/data/ImportKey.java) class to take the key and cert and place it in a newly constructed JKS keystore.
+I modified the ImportKey java source to use the keystore password changeit and to use the key alias importkey and to save the resulting keystore in the file opennms.keystore
+`
+java ImportKey key.der cert.der
+`
+
+== Using wildcard certifcate with a Pre-existing Private Key and Certificate
+
+Description from: http://www.agentbob.info/agentbob/79-AB.html?branch=1&language=1
+
+Copied wildcard certifcate to /home/monitor/ssl
+
+[source, console]
+----
+root@opennms: /home/monitor/ssl# ls -al
+
+-rw-r--r--  1 root    root     5626 Jun 24 11:09 cabundle.crt
+-rw-r--r--  1 root    root     1363 Sep 15 11:58 cert.der
+-rw-r--r--  1 root    root     1903 Jun 24 11:09 certificate.crt
+-rw-r--r--  1 root    root     1704 Jun 24 11:09 certificate.key
+-rw-r--r--  1 root    root     1679 Sep 15 09:24 certificate.new.key
+-rw-r--r--  1 root    root     7256 Jun 24 11:09 certificate.p7b
+-rw-rw-r--  1 monitor monitor  3281 Sep 15 11:53 ImportKey.class
+-rw-r--r--  1 root    root     1218 Sep 15 11:58 key.der
+-rw-r--r--  1 root    root     1423 Sep 15 09:32 keystore.jks
+----
+
+ImportKey.class download:
+http://www.agentbob.info/agentbob/81/version/default/part/AttachmentData/data/ImportKey.class
+
+Convert both, the key and the certificate into DER format using openssl
+
+[source, console]
+----
+openssl pkcs8 -topk8 -nocrypt -in key.pem -inform PEM -out key.der -outform DER
+openssl pkcs8 -topk8 -nocrypt -in certificate.key -inform PEM -out key.der -outform DER
+openssl x509 -in certificate.crt -inform PEM -out cert.der -outform DER
+
+java ImportKey key.der cert.der
+root@opennms:/home/monitor/ssl# java ImportKey key.der cert.der
+Using keystore-file : /root/keystore.ImportKey
+One certificate, no chain.
+Key and certificate stored.
+Alias:importkey  Password:importkey
+root@opennms:/home/monitor/ssl#
+----
+
+Now we have a proper JKS containing our private key and certificate in a file called keystore.ImportKey, using 'importkey' as alias and also as password
+
+Copy /root/keystore.ImportKey to /ect/opennms/opennms.keystore
+
+Change password importkey to changeit (this password is also in opennms.properties file, section https)
+`
```

Comment:
It looks like there are some markdown leftovers for the console in- and outputs.